server/src/main/scala/controllers/UsersController.scala

```diff
@@ -25,6 +25,15 @@ class UsersController @Inject() (
     } yield Ok(Json.toJson(response))
   }
 
+  def verifyEmail() = handleJsonBody[VerifyEmailRequest] { request =>
+    val body = request.body
+    val (userId, token) = decodeAWSToken(body.token)
```

Comment:
`decodeAWSToken` throws exceptions, you need to wrap those in a `Future`, like `Future.fromTry(Try(decodeAWSToken(...))`